flowpipe/node.py

```diff
@@ -107,10 +111,6 @@ def evaluate(self):
                 self.file_location, self.class_name,
                 json.dumps(self._sort_plugs(inputs), indent=2)))
 
-        inputs = {}
-        for name, plug in self.inputs.items():
-            inputs[name] = plug.value
-
         # Compute and redirect the output to the output plugs
         start_time = time.time()
         outputs = self.compute(**inputs) or dict()
```

Comment:
Question:
With the omission of the explicit assembly of a dictionary of input values, do we not send a list of `Plug` instances to the compute function instead of unwrapping a dictionary of values?

Reply:
Hi @neuneck :slightly_smiling_face: ,
I think the git diff might have failed here, if you view the file you will see this code is still present at lines 105-107.